### DISCUSSION TITLE
Unexpected high percentage of Shell

### DISCUSSION THREAD

**Discussion by KevinnZou** [github-linguist/linguist#6594] [Classification]
My library is built using Kotlin and consists of only one shell file with 19 lines. Surprisingly, the shell file accounts for 72% of the library, while Kotlin only takes up 28%. I am unable to understand why this has happened.
https://github.com/KevinnZou/compose-multiplatform-library-template
![截屏2023-10-27 17 34 21](https://github.com/github-linguist/linguist/assets/43654611/493dcba6-8cab-43aa-9ac1-a6f3dfe2d5b2)

**Comment by lildude** [comment]
This will be because Linguist ignores all files under `samples` by default. You’ll need to use an override to mark them as not documentation.

**Reply by KevinnZou** [reply]
Thanks for your reply! Could you give an example on how to mark `sample` as not documentation?

**Reply by lildude** [reply]
https://github.com/github-linguist/linguist/blob/master/docs/overrides.md
